packages/service/ni_measurement_plugin_sdk_service/_internal/parameter/encoder.py

```diff
@@ -48,6 +48,7 @@ def serialize_parameters(
             elif parameter_metadata.type == FieldDescriptorProto.TYPE_MESSAGE:
                 getattr(message_instance, field_name).CopyFrom(parameter)
             else:
+                print(field_name, repr(parameter))
```

Comment:
Is there a particular reason to print the field name and the parameter, here? Or was it included for debugging?

Reply:
Thanks for catching that. That was debugging code.